test/models/measurement_test.rb

```diff
@@ -1,7 +1,12 @@
 require 'test_helper'
 
 class MeasurementTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'should scope measurements per sensor' do
+    create_list(:measurement, 3, sensor: create(:sensor))
+    create_list(:measurement, 3, sensor: create(:sensor))
+    create_list(:measurement, 3, sensor: create(:sensor))
+
+    measurements = Measurement.last_per_sensor(2).to_a
+    assert_equal(6, measurements.count)
```

Comment:
It would probably be interesting to test the edge cases, e.g. what happens if there are less measurements than the limit. Or the combination of multiple filters (e.g. last_per_sensor + last).

Ordering is irrelevant in my opinion :)